cowbird/monitoring/monitoring.py

```diff
@@ -34,8 +40,8 @@ def __init__(self, config=None):
                        must be initialized with a proper config and after that the init function should not be hit.
         """
         if not config:
-            raise Exception("Without proper application settings, the Monitoring class cannot obtains a proper database"
-                            " store.")
+            raise MonitoringConfigurationException("Without proper application settings, the Monitoring class cannot "
```

Comment:
You can add `if not config:  # pragma: no cover` here. 
It's ok for this case since it only early-raises a more specific error that is expected to happen later.